src/lib/scroller.js

```diff
@@ -80,34 +88,61 @@ Scroller.prototype.requestRefresh = function(){
 Scroller.prototype._renderVisible = function(){
   this.animateRequest = null;
   if(this.dirty && this.console){
-    var top = this.console.scrollTop;
+    const top = this.console.scrollTop;
     if(this.sticky){
-      this.startPosition = Math.max(0, this.lines.length - this.minVisible);
+      this.startPosition = Math.max(this.lineOffset, this.lineCount() - this.visibleCount);
+      this.endPosition = this.lineCount();
```

Comment:
define this above `this.startPosition` so you don't have to call `this.lineCount()` twice and then use `this.endPosition` in the `this.startPosition` calculation.